src/clj/witan/send/domain/academic_years.clj

```diff
@@ -93,6 +93,9 @@
 (def secondary-school
   (into key-stage-3 key-stage-4))
 
+;; Key Stages 1 to 5
+(def school-age (reduce #(into %1 %2) (sorted-set) [primary-school secondary-school key-stage-5]))
```

Comment:
this is handy. thx

Reply:
@seb231 I'd like to merge this bit even if we don't merge the rest but move it to another library